src/main/java/frc/robot/constants/RobotConstants.java

```diff
@@ -36,17 +37,21 @@ public static final class DriveConstants {
     public static final int kGyroId = 15;
 
     // Chassis configuration
-    public static final double kTrackWidth = Units.inchesToMeters(26.5);
+    public static final Measure<Distance> kTrackWidth = Units.Inches.of(26.5);
```

Comment:
Is 26.5 being recorded as inches in kTrackWidth

Reply:
yes, though something more accurate than holding a tape measure up to the drive train may be in order